docs/reference/pages/model_reference.rst

```diff
@@ -356,6 +386,89 @@ Methods and properties
 
     .. automethod:: get_site_root_paths
 
+
+Locale
+======
+
+The ``Locale`` model defines the set of languages and/or locales that can be used on a site.
+Each ``Locale`` record corresponds to a "language code" defined in the :ref:`wagtail_content_languages_setting` setting.
+
+Wagtail will initially set up one ``Locale`` to act as the default language for all existing content.
+This first locale will automatically pick the value from ``WAGTAIL_CONTENT_LANGUAGES`` that most closely matches the site primary language code defined in ``LANGUAGE_CODE``.
+If the primary language code is changed later, Wagtail will **not** automatically create a new ``Locale`` record or update an existing one.
+
+Before internationalisation is enabled, all pages use this primary ``Locale`` record.
+This is to satisfy the database constraints. And it makes it easier to switch internationalisation on at a later date.
+
+Changing ``WAGTAIL_CONTENT_LANGUAGES``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Languages can be added or removed from ``WAGTAIL_CONTENT_LANGUAGES`` over time.
+
+Before removing an option from ``WAGTAIL_CONTENT_LANGUAGES``, it's important that the ``Locale``
+record is updated to a use a different content language or is deleted.
+Any ``Locale`` instances that have invalid content languages are automatically filtered out from all
+database queries making them unable to be edited or viewed.
+
+Methods and properties
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: Locale
+    :noindex:
+
+    .. attribute:: language_code
+
+        Foo
+
+    .. automethod:: get_default
+
+    .. automethod:: get_active
+
+    .. automethod:: get_display_name
+
+
+Translatable Mixin
+==================
+
+``TranslatableMixin`` is an abstract model that can be added to any non-page Django model to make it translatable.
+Pages already have this mixin added by Wagtail, so no need to add it to them.
+
+Methods and properties
+~~~~~~~~~~~~~~~~~~~~~~
+
+The ``locale`` and ``translation_key`` fields have a unique key constraint to prevent the object being translated into a language more than once.
+
+.. class:: TranslatableMixin
+    :noindex:
+
+    .. attribute:: locale
+
+        (Foreign Key to :class:`~wagtail.core.models.Locale`)
+
+         For pages, this defaults to the locale of the parent page.
+
+    .. attribute:: translation_key
+
+        (uuid)
+
+        A UUID that is randomly generated whenever a new model instance is created.
+        This is shared with all translations of that instance so can be used for querying translations.
+
+    .. automethod:: get_translations
+
+    .. automethod:: get_translation
+
+    .. automethod:: get_translation_or_none
+
+    .. automethod:: has_translation
+
+    .. automethod:: copy_for_translation
+
+    .. automethod:: get_translation_model
```

Comment:
The docstring here says:
> Gets the model which manages the translations for this model. (The model that has the “translation_key” and “locale” fields) Most of the time this would be the current model, but some sites may have intermediate concrete models between wagtailcore.Page and the specfic page model.

The mention of intermediate concrete models doesn't seem right - surely _any_ subclass of Page would be an example of a model where this doesn't return the current model? (Also, best to avoid the word "site" when not talking about Site records - usually I'd say project or installation.)

Perhaps this should be something like: "Typically this would be the current model, but it may be a superclass when multi-table inheritance is in use (as is the case for wagtailcore.Page)."

Reply:
Fixed, thanks!